resources/default.rb

```diff
@@ -0,0 +1,133 @@
+property :link_name, String, name_property: true
+property :link, String
+property :path, String
+property :priority, [String, Integer]
+
+action :install do
+  alternatives_install
+end
+
+action :set do
+  alternatives_set
+end
+
+action :remove do
+  alternatives_remove
+end
+
+action :auto do
+  alternatives_auto
+end
+
+action :refresh do
+  alternatives_refresh
+end
```

Comment:
the helper functions here aren't useful at all, a method that calls a method just to bury the implementation details further down the page detracts from rather than adds to readability.

Reply:
Merged back into the actions